api/src/main/java/org/pmiops/workbench/compliance/ComplianceService.java

```diff
@@ -22,5 +24,16 @@
    * @return list of badges/completed training by user
    * @throws ApiException
    */
-  List<BadgeDetails> getUserBadge(int userMoodleId) throws ApiException;
+  List<BadgeDetailsDeprecated> getUserBadge(int userMoodleId) throws ApiException;
```

Comment:
Why did we move to the deprecated class here? Should this method be marked deprecated as well?